bin/acfg1.py

```diff
@@ -17,7 +17,7 @@
 METADATA_FILE_NAME = "acfg.metadata"
 
 
-def copy_tree(source, destination, dryrun=False):
+def copy_tree(source, destination, hostclasses=[], dryrun=False):
```

Comment:
Actually, should probably be `None`, shouldn't it?  Having a mutable object as a default is frowned upon.